src/main/resources/assets/forge/lang/en_US.lang

```diff
@@ -24,7 +24,7 @@ forge.configgui.biomeSkyBlendRange.tooltip=Control the range of sky blending for
 forge.configgui.biomeSkyBlendRange=Biome Sky Blend Range
 forge.configgui.clumpingThreshold.tooltip=Controls the number threshold at which Packet51 is preferred over Packet52.
 forge.configgui.clumpingThreshold=Packet Clumping Threshold
-forge.configgui.disableVersionCheck.tooltip=Set to true to disable Forge's version check mechanics. Forge queries a small json file on our server for version information. For more details see the ForgeVersion class in our github.
+forge.configgui.disableVersionCheck.tooltip=Set to true to disable Forge's version check mechanics. Forge queries a small json file on our server for version information. For more details see the ForgeVersion class in our Github.
```

Comment:
If you're changing the capitalisation of "github" here, capitalise properly: GitHub